### PR TITLE
bug fixed

### PR DESCRIPTION
**Pull request by neera11**
please review

### CODE REVIEW

File: public/js/blocks.js [hunk 1]
```diff
@@ -301,10 +301,9 @@ class Edit extends Component {
           src: "https://cdn.razorpay.com/static/widget/subscription-button.js",
           "data-plugin": "wordpress_"+razorpay.payment_buttons_plugin_version,
           "data-subscription_button_id": buttonId,
```
Comment (ChetanGN) [inline public/js/blocks.js:303]: @neera11 What is the use of this?

Comment (ChetanGN) [inline public/js/blocks.js:303]: @neera11 why you had removed this? Any issue ?

Comment (neera11) [reply public/js/blocks.js:303]: by default passing button theme , was creating problem

**Review by ChetanGN — APPROVED** [review]
LGTM

**Pull request merged**